embedded-database-spring-test/src/main/resources/META-INF/spring-configuration-metadata.json

```diff
@@ -130,8 +130,8 @@
     {
       "name": "zonky.test.database.postgres.yandex-provider.postgres-version",
       "type": "java.lang.String",
-      "description": "Version of EnterpriseDB PostgreSQL binaries (https://www.enterprisedb.com/download-postgresql-binaries).",
-      "defaultValue": "11.10-1"
+      "description": "Version of EnterpriseDB PostgreSQL binaries (https://www.enterprisedb.com/download-postgresql-binaries). 11+ for Linux available on https://www.postgresql.org/download/",
+      "defaultValue": "10.23-1"
```

Comment:
The PostgreSQL versions for OpenTable and Yandex providers must match the other providers, regardless of whether they support Linux platform or not. I plan to remove them in the next major version anyway.